File: PHP/_res/Student.php
<?php
/**
 * @file   : Student.php
 * @time   : 22:05
 * @date   : 2021/10/5
 * @emailto: 9727005@qq.com
 * @creator: ShanDong Xiedali
 * @company: HiLand & RainyTop
 */


namespace PHP\Study\_res;

class Student
{
    /**
     * @var string
     */
    private string $name;

    /**
     * @var int
     */
    private int $age;


    public function __construct($name,$age)
    {
        $this->age=$age ;
        $this->name=$name;
    }

    public function getName(): string
    {
        return $this->name;
    }
}

File: Pylint.Study/utils/student.py
"""
 * @file   : student.py
 * @time   : 7:53
 * @date   : 2024/4/5
 * @mail   : 9727005@qq.com
 * @creator: ShanDong Xiedali
 * @company: HiLand & RainyTop
"""


class Student(object):
    """ """

    def __init__(self, name: str):
        self.name = name

    def say_hello(self, message: str = "你好"):
        return f"{self.name},{message}"
